modules/aws-cloudwatch.adoc

```diff
@@ -6,7 +6,7 @@
 
 = Amazon CloudWatch
 
-Amazon CloudWatch forwards logs from {product-title} (ROSA) to the AWS console for viewing. You must first install the ROSA `cluster-logging-operator` using the `rosa` CLI before installing the Amazon CloudWatch service through the OpenShift Cluster Manager (OCM) console.
+Amazon CloudWatch forwards logs from {product-title} (ROSA) to the AWS console for viewing. You must first install the ROSA `cluster-logging-operator` using the `rosa` CLI before installing the Amazon CloudWatch service through the OpenShift Cluster Manager console.
```

Comment:
I haven't seen it referred to as "OpenShift Cluster Manager console" so I think it would be more consistent (and searchable) as simply "through OpenShift Cluster Manager"